modules/manage/pages/manage-security/configure-ldap.adoc

```diff
@@ -438,18 +475,18 @@ To configure an LDAP server to be used as a point of authentication for Couchbas
 --password password \
 --hosts 172.23.124.25 --port 389 \
 --encryption startTLS \
---ca-cert '/path/to/cert' \
---query-dn 'cn=admin,dc=example,dc=com' \
---query-pass 'password' \
+--ldap-cacert '/path/to/cert' \
+--user-dn-query 'cn=admin,dc=example,dc=com' \
+--bind-password 'password' \
 --authentication-enabled 1 \
---user-dn-mapping '{"template":"uid=%u,ou=users,dc=example,dc=com"}' \
+--user-dn-template '{"template":"uid=%u,ou=users,dc=example,dc=com"}' \
 --authorization-enabled 1 \
 --group-query 'ou=groups,dc=example,dc=com??one?(member=%D)'
 ----
 
 This call references the LDAP server at `172.23.125.25`, on port `389`, and specifies `--authorization-enabled` and `--authentication-enabled` for the user-credentials that will be passed from Couchbase Server.
 The argument specified for `--group-query` is the query that retrieves the LDAP groups of which the user is a member.
-A `--user-dn-mapping` is specified as a distinguished name template.
+A `--user-dn-template` is specified as a distinguished name template.
```

Comment:
Please also update the 'couchbase-cli-setting-ldap' page.
 --user-dn-mapping option is removed.
Instead we now have two options
`--user-dn-template` and `--user-dn-query`
You can take description for those options from CLI's help

Reply:
Looks like this has been handled by Patrick's team: https://docs.couchbase.com/server/6.5/cli/cbcli/couchbase-cli-setting-ldap.html